source/routing/raptor_utils.h

```diff
@@ -98,6 +112,13 @@ struct Labels {
     IdxMap<type::StopPoint, DateTime> dt_pts;
     // At what time wan we reach this label with a transfer
     IdxMap<type::StopPoint, DateTime> dt_transfers;
+
+    // walking_duration_pts[stop_point] stores the walking duration elapsed during the whole
+    // journey from the starting point which reaches stop point at DateTime dt_pts[stop_point] with public transport
+    IdxMap<type::StopPoint, DateTime> walking_duration_pts;
+    // walking_duration_pts[stop_point] stores the walking duration elapsed during the whole
```

Comment:
```suggestion
    // walking_duration_pts[stop_point] stores the walking transfer duration elapsed during the whole
```
no?

Reply:
No, this is the same as what's done on dt_pts and dt_transfers, but for "fallback" duration (fallback + transfers if I'm correct).

See `PT1` and `TR1` in doc: https://github.com/CanalTP/navitia/blob/dev/documentation/rfc/doc-routing-algorithm.md


```suggestion
    // walking_duration_pts[stop_point] stores the fallback+transfer duration elapsed during the whole
```
(same above)

Reply:
For now it only stores the fallback part.
I am testing adding the transfer part, but it has effects on the resulting journeys. 
Not sure yet if it should be just fallback or fallback + transfer.
I added a more precise description comments in 051ab1a4d1bf54e95d2a5d195910e797830ad317